### PR TITLE
Krypto

### PR DESCRIPTION
**Pull request by florian-sabonchi**
In this commit, the cog CryptoCurrenciesCog was added, which lists the last three price of cryptocurrencies listed on CoinMarketCap above

### CODE REVIEW

**Review by felbinger — CHANGES_REQUESTED** [review]
- move `morpheushelper/models/coinmarketcap.py` from `models` to cog.
- refactor docker workflow (reset to original version)

**Comment by luwol03** [comment]
I would suggest to create an extra folder e.g. called `api_models` and put the clever bot api class and the i run code api class 
from #130 there as well to get a clean folder structure because the cogs are imao only for discord classes.

**Comment by FlorianJSa** [comment]
> I would suggest to create an extra folder e.g. called `api_models` and put the clever bot api class and the i run code api class
> from #130 there as well to get a clean folder structure because the cogs are imao only for discord classes.

This would be a useful folder structure.

File: morpheushelper/coinmarketcap.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            'limit': '3',
+            'convert': 'USD'
+        }
+
+        headers = {
+            "X-CMC_PRO_API_KEY": self.token,
+            'Accepts': 'application/json'
+        }
+
+        return requests.get(url=f"{self.URL}/v1/cryptocurrency/listings/latest", params=parameters, headers=headers)
```
Comment (Paul-Brauckmann) [inline morpheushelper/coinmarketcap.py:22]: sollte entsprechend auch Asynchron stattfinden.

File: morpheushelper/coinmarketcap.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import requests
+
+
+class CoinMarketCap:
+    URL = "https://pro-api.coinmarketcap.com"
+
+    def __init__(self, token):
+        self.token = token
+
+    def get_cryptocurrencies_list(self):
```
Comment (Paul-Brauckmann) [inline morpheushelper/coinmarketcap.py:10]: Die Methode sollte Async sein und im Zweifel nicht den Thread blockieren….

**Review by ghost — CHANGES_REQUESTED** [review]
Due to a the restructuring of the Bot this pull reuest is not up to date. Feel free to close this and reopen it in the [cogs-repo](https://github.com/PyDrocsid/cogs).

**Pull request closed without merging**